src/bootstrap-table.js

```diff
@@ -307,6 +307,9 @@
     paginationDetailHAlign: 'left', // right, left
     paginationPreText: '&lsaquo;',
     paginationNextText: '&rsaquo;',
+    paginationSuccessivelySize: 3, // Maximum successively number of pages in a row
```

Comment:
I think here is better to be `5` by default, which is consistent with the previous usage habits.